### DISCUSSION TITLE
Support for jellyfin collections

### DISCUSSION THREAD

**Discussion by dDiverVS** [arabcoders/watchstate#339] [General] [answered]
Hi, I am having some issues with collections on jellyfin, it seems to be that there is a specific content type "boxset" within jellyfin which groups movies into collections when you enable the option "Group movies into collections" in the display tab.

This is causing watchstate to not identify the movies within those collections and therefore not update the play state in jellyfin, I found a workaround wich is disabling this setting but I really don't like seeing all movies instead of collections in my dashboard, is there any plan to support this feature in the future?

Thanks!

**Comment by arabcoders** [comment, marked as answer]
@dDiverVS as far as i know for movies to be in collection it has to be in a library to begin with, and when movie is marked as played in a collection it does so for the library as well. thus you can ignore the boxset libraries and the state will be synced. Can you confirm with logs that is not the case? i think i had to check this as well as it was requested before and it turned out the movie does show up in the library

**Reply by dDiverVS** [reply]
Yes, I will retest it tomorrow and share the logs, the issue only happens if you enable the specific option "Group movies into collections" in dashboard - libraries - display, if you disable it it works fine.
The setting "Automatically add to collection" does not affect watchstate, I think the reason is that when you enable the "Group movies into collections" setting the movies are no longer displayed individually in the library at all, if you go into your library you will see the collection even in the movies tab.

**Reply by arabcoders** [reply]
As far as i know even though jellyfin hides the movies from the library and instead only show them in the collection or boxset it's a UI thing, they are still show up in the API request. If you can run an import and check `db:list` for the movies that are in boxset i think you will see it there. If that not the case, Please hop on discord so we can chat about possible ways to support it.

**Reply by arabcoders** [reply]
@dDiverVS did you have chance to try what i suggested?

**Reply by dDiverVS** [reply]
Sorry life got a bit into the way, I will try it in a couple of hours and report back :)

**Reply by dDiverVS** [reply]
Ok, I don't know what happened as I first tested it as there was no way to get some of my movies to be in sync (even with syncing ignoring the date).
After reimporting both my servers everything seems to work fine, thanks!

**Reply by arabcoders** [reply]
No worries. glad everything is working for you.
